lerna.json

```diff
@@ -1,19 +1,20 @@
 {
   "$schema": "node_modules/@lerna-lite/cli/schemas/lerna-schema.json",
-  "useWorkspaces": true,
   "version": "3.3.4",
   "exact": true,
   "changelog": false,
   "ignoreChanges": [
     "**/test/**",
     "**/*.md"
   ],
+  "npmClient": "npm",
   "private": false,
   "push": true,
   "command": {
     "version": {
       "allowBranch": [
-        "main"
+        "main",
```

Comment:
is this needed? (I mean, it will propagate to perps-andromeda branch when merged)